content/_help/verify-your-identity/verify-your-identity-in-person/find-a-participating-post-office._en.md

```diff
@@ -3,7 +3,6 @@ title: Find a Participating Post Office to finish identity verification
 child: true
 order: 1
 permalink: /help/verify-your-identity/verify-your-identity-in-person/find-a-participating-post-office/
-exclude: true
```

Comment:
Shifting this to the config file allows the value to be changed via a CLI option instead of relying purely on hardcoded values. Not my favorite thing to duplicate the `defaults` array, but this option still seems better.